docs/Tutorials/Basics/simple-drops.md

```diff
@@ -0,0 +1,280 @@
+---
+sidebar_label: 'Simple Drop'
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Simple Drop
+
+## Introduction
+In this tutorial, you will learn how to create a Simple $NEAR Drop. This will allow you to onboard users using a simple Web2 style link. Alternatively, if the user has a wallet already, the assets can be sent there instead.
+
+In a simple drop, the assets consist of $NEAR. 
+
+
+<p align="center"> <img src={require("/static/img/docs/basic-tutorials/simple/nw-claim.png").default} alt="Near Wallet claim" width="100%"/> </p>
+
+To learn more about the Simple Drop, see the [concepts page](../../Concepts/Keypom%20Protocol/Github%20Readme/Types%20of%20Drops/simpledrops.md)
+
+---
+
+## Prerequisites
```

Comment:
I think in this section we should bootstrap the user fully:
1. Create a folder where the project will live.
2. Create a default `package.json` file
3. Create an empty `simple-keypom.js` file.
4. Install keypom SDK using `npm install keypom-js`

This is truly the prerequisites and I think we should add this process in ^